src/aspire/source/__init__.py

```diff
@@ -7,13 +7,15 @@
 
 from aspire.image import Image
 from aspire.io.starfile import save_star
-from aspire.source.xform import (Downsample, FilterXform, LinearIndexedXform,
-                                 LinearPipeline, Multiply, Pipeline, Shift)
+
+from aspire.source.xform import (Downsample, FilterXform, FlipXform, LinearIndexedXform,
+                                 LinearPipeline, Multiply, Pipeline, Reduce, Shift)
 from aspire.utils import ensure
 from aspire.utils.coor_trans import grid_2d
-from aspire.utils.filters import MultiplicativeFilter, PowerFilter
+from aspire.utils.filters import (MultiplicativeFilter, PowerFilter)
```

Comment:
Why is this changing? Maybe we need to run `isort` here to make sure things stay consistent.

Reply:
Made change to be consistent with the format after running `isort.`

Reply:
Are you sure you ran `isort` on this file and it keeps the parentheses? I tried running it and it restored the original line (without parentheses).

Reply:
If it helps, we don't need to worry about making changes intentionally for isort right now. That is something we can run systematically later. We got the bulk of them.

Reply:
I didn't run isort. The reason I did this is that many previous import statements were included parentheses after a major  PR of isort changing from the `develop` branch. Probably only need the parentheses if there are more than three functions. I will restore it.